public/modules/users/config/users.client.routes.es6

```diff
@@ -19,7 +19,8 @@ angular.module('users').config(['$stateProvider', function ($stateProvider) {
 		templateUrl: 'modules/users/views/authentication/signup.client.view.html'
 	}).state('signin', {
 		url: '/signin',
-		templateUrl: 'modules/users/views/authentication/signin.client.view.html'
+		//templateUrl: 'modules/users/views/authentication/signin.client.view.html'
```

Comment:
Do not use comments! You have version control system! You don't need save commented lines!